GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -354,6 +354,10 @@ private void LoadNodeToolTips(Nodes loadedNodes, CancellationToken token)
                         catch (OperationCanceledException)
                         {
                         }
+                        //// Comment out to debug BugReporter
+                        ////catch (GitExtUtils.ExternalOperationException)
+                        ////{
+                        ////}
```

Comment:
Please revert back

Reply:
I would like to have a comment with how to debug the BugReporter when the debugger is in use
What do you suggest?
Is the commented out code bothering you?

Reply:
Keeping when squash merging in about a day unless clarified